modules/ROOT/pages/design-assigning-permissions.adoc

```diff
@@ -0,0 +1,49 @@
+= Set User Access Permissions
+
+Design Center provides multiple ways to set up access management to provide collaboration across teams. 
+
+Depending on your organization's permission structure, you can assign permissions using the xref:access-management::teams.adoc[Teams] feature, the xref:access-management::users.adoc#grant-user-permissions[Roles] feature (deprecated), or by assigning an xref:access-management::users.adoc#grant-user-permissions[individual user permission].
+
+NOTE: Mule Applications can't be shared.
```

Comment:
This NOTE should be posted in the "Sharing Projects from Design Center" section.